tests/braille-specs/ja-kantenji.yaml

```diff
@@ -10,6 +10,25 @@ table:
   dots: 8
 tests:
 
+# Removed implementations for the computer braille in Japanese braille system,
+# because this causes NVDA and iOS/Mac OS VoiceOver crash.
+# Following lines are removed from ja-kantenji.utb
+# begcomp 8-378
+# endcomp 8-678
+# compbrl ://
+# compbrl ()
+# compbrl www.
+# compbrl ::
+# compbrl .jp
+# compbrl .com
+# ...
```

Comment:
This explanation can be dropped from the test file. The test file should represent what the Kantenji braille specification dictates. It shouldn't contain implementation details.

Also the "not with the computer braille rule--enclosed between opening dots 8-378 and ending dots 8-678" part can be dropped. Unless that is what the Kantenji braille specification says. In that case the test should be changed and should be flagged with `xfail` to indicate that the Liblouis table does not behave according to the official rules.

Reply:
I deleted the computer braille indicator part, because this rule is borrowed from the other Japanese Kana Braille system(the mainstream one), and not the rule of the official Braille Kanji system.